python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -25,45 +25,45 @@ alpha_k : 2-d sequence of `float`
     The coefficients of the functional chaos expansion.
 Psi_k : sequence of :class:`~openturns.Function`
     The functions of the multivariate basis selected by the algorithm.
-residuals : sequence of `float`, :math:`\hat{\vect{r}} \in \mathbb{R}^{n_Y}`
+residuals : sequence of `float`, :math:`\hat{\vect{r}} \in \Rset^{\physicalOutputDimension}`
     For each output component, the residual is the square root of the sum of 
     squared differences between the model and the meta model, divided by the 
     sample size.
-relativeErrors : sequence of `float`, :math:`\widehat{\vect{re}} \in \mathbb{R}^{n_Y}`
+relativeErrors : sequence of `float`, :math:`\widehat{\vect{re}} \in \Rset^{\physicalOutputDimension}`
     The relative error is the empirical error divided by the 
     sample variance of the output.
 
 Notes
 -----
-Let :math:`n \in \mathbb{N}` be the sample size. 
-Let :math:`n_Y \in \mathbb{N}` be the dimension of the output of the 
+Let :math:`\sampleSize \in \mathbb{N}` be the sample size. 
+Let :math:`\physicalOutputDimension \in \mathbb{N}` be the dimension of the output of the 
 physical model. 
-For any :math:`j = 1, ..., n` and any :math:`i = 1, ..., n_Y`, 
-let :math:`y_{j, i} \in \mathbb{R}` be the output of the physical 
-model and let :math:`\hat{y}_{j, i} \in \mathbb{R}` be the output of the 
+For any :math:`j = 1, ..., \sampleSize` and any :math:`i = 1, ..., \physicalOutputDimension`, 
+let :math:`y_{j, i} \in \Rset` be the output of the physical 
+model and let :math:`\hat{y}_{j, i} \in \Rset` be the output of the 
 metamodel. 
-For any :math:`i = 1, ..., n_Y`, let :math:`\vect{y}_i \in \mathbb{R}^n` be 
-the sample output and let :math:`\hat{\vect{y}}_i \in \mathbb{R}^n` be the 
+For any :math:`i = 1, ..., \physicalOutputDimension`, let :math:`\physicalOutputRandomVector_i \in \Rset^\sampleSize` be 
+the sample output and let :math:`\widehat{\physicalOutputRandomVector}_i \in \Rset^\sampleSize` be the 
```

Comment:
I try to answer on the two different points.
- The index j is the one in the set {1, ..., n} where n is the sample size. Therefore it cannot be \physicalOutputRandomVector_j and \widehat{\physicalOutputRandomVector}_j because these are the j-th observed output of the model and the metamodel. These variables do not exist, because the output has dimension $n_y$. Therefore, the i-th component of the j-th output of the model is $Y_{i, j}$, not $Y_j$ (this is not easy to see, because of a bug in github see [github/markup](https://github.com/github/markup/issues?q=is%3Aissue+is%3Aopen+latex+)).
- On the second topic, I do not like your suggestion in this particular context because a componentwise operator that takes only a single input is clear (e.g. log or sqrt), but an operator which has several inputs can be interpreted in different ways. More importantly, the code I wrote is questionable in my opinion, in the sense that the equations should be written in `MetaModelResult` instead instead of duplicating them in its children classes. The problem though, is that the doc and code of `MetaModelResult` should be improved, as shown in #2455 and #2292.

Meanwhile, I suggest to leave this content as is and to focus on the `MetaModelResult` instead. Do you agree on this strategy?

Reply:
OK for me.